.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+<!--
+    Thanks for submitting a Pull Request (PR) to this project.
+    Your contribution to this project is greatly appreciated!
+
+    Please make sure the PR title is short but a descriptive summary of the PR,
+    e.g "Add common test to test the length of the relative file path".
+
+    You may remove this comment block, and the other comment blocks,
+    but please keep the headers and the task list.
+-->
+#### Pull Request (PR) description
+<!--
+    Replace this comment block with a description of your PR.
+-->
+
+#### This Pull Request (PR) fixes the following issues
+<!--
+    Replace this comment block with the list of issues or n/a.
+    Use format:
+    Fixes #123
+    Fixes #124
+-->
+
+#### Task list
+<!--
+    To aid community reviewers in reviewing and merging your PR, please take
+    the time to run through the below checklist and make sure your PR has
+    everything updated as needed.
+
+    Change to [x] for each task in the task list that applies to your pull
+    request (PR). For those task that don't apply to you pull request (PR),
+    leave those as is.
+-->
+- [ ] Added an entry under the Unreleased section in the CHANGELOG.md. Entry
```

Comment:
Should this be README.md rather than CHANGELOG.md? Looks like the change log is in the README?